World Cup Script.py

```diff
@@ -8,30 +9,33 @@
 matchesPath = 'F:/World Cup/2018 - Mania 7K/Stream/Scripts/Matches/'
 applicationsPath = 'F:/World Cup/2018 - Mania 7K/Stream/Scripts/Applications/'
 
+
 def countryName(color, country):
 
     # Creates the country.txt files
-    countryFile = open(scriptsPath + 'country_file_' + color + '.txt', 'w')
+    countryFile = open('{0}country_file_{1}.txt'.format(scriptsPath, color), 'w')
 
     # Writes the user input into the above .txt
     countryFile.write(country)
 
     # Closes the file

```

Comment:
Probs you can slo simplify docs here. put a summary  of what the methods does on the first comment instead.

Reply:
I went ahead and changed everything to f strings myself to make sure I understand the concept :) Ended up removing the program opening function as a whole because it didn't prove useful

Reply:
I'm surprised I forgot about f-strings instead of `string.format()` lol. It would've made everything look a lot cleaner.
And with stuff like `programsOpen()` it's better to put everything in a list so you don't have to write the same line over and over. In general, it just makes your code look a lot cleaner and it's less painful on your part :P